src/bentoml/models.py

```diff
@@ -227,6 +227,48 @@ def pull(
     return _cloud_client.pull_model(tag, force=force)
 
 
+if t.TYPE_CHECKING:
+
+    class CreateKwargs(t.TypedDict):
+        module: str
+        api_version: str | None
+        signatures: t.Required[ModelSignaturesType]
+        labels: dict[str, t.Any] | None
+        options: ModelOptions | None
+        custom_objects: dict[str, t.Any] | None
+        external_modules: t.List[ModuleType] | None
+        metadata: dict[str, t.Any] | None
+        context: t.Required[ModelContext]
+        _model_store: t.NotRequired[ModelStore]
+
+
+@t.overload
+@contextmanager
+def create(
+    name: Tag | str, /, **attrs: t.Unpack[CreateKwargs]
+) -> t.Generator[Model, None, None]:
+    ...
```

Comment:
oh it supports `**kwargs`:
```python
attrs = dict(module="bentoml.transformers", context=ModelContext())

with bentoml.models.create("name", **attrs): # type-safe
  ...
```

where this will fail:
```python
attrs = dict(module="bentoml.transformers", not_a_valid_arg="asdf")
with bentoml.models.create("name, **attrs): # <- should show error with type check
  ...
```

Reply:
Ah, nice! I'm surprised that it doesn't work normally...

Reply:
Oh this is a new feature type-safe https://docs.python.org/3.13/library/typing.html#typing.Unpack

Reply:
No, I mean that it's strange that `Unpack` typing is required at all for `**attrs` to work.